extra_sdk_options.go

```diff
@@ -0,0 +1,34 @@
+package conductoroneapi
+
+import (
+	"net/url"
+	"strings"
+)
+
+const GoSDKClientID = "2RCzHlak5q7CY14SdBc8HoZEJRf"
```

Comment:
weird exported const name.  maybe lets start without exporting it?

at some level would expect it to be ClientIdGolangSDK? but dunno.

Reply:
It's already pretty hard to use our SDK considering they need to do all the funkyness with token source, wouldn't removing this export just make it more difficult?